library/hitch

```diff
@@ -1,12 +1,8 @@
-# this file was generated using https://github.com/varnish/docker-hitch/blob/3f7f06a4eeb066e03f81f5e3d170d1e13606e69e/populate.sh
+# this file was generated using https://github.com/varnish/docker-hitch/blob/3fe55c4296018b04c7e1d2efdfcadce9c4017e0a/populate.sh
 Maintainers: Thijs Feryn <thijs@varni.sh> (@thijsferyn),
              Guillaume Quintard <guillaume@varni.sh> (@gquintard)
 GitRepo: https://github.com/varnish/docker-hitch.git	
 
-Tags: 1, 1.7, 1.7.3, 1.7.3-1, latest
+Tags: 1, 1.8, 1.8.0, 1.8.0-1, latest
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
```

Comment:
FYI, this should either be updated to be `FROM debian:bookworm-slim` or should drop `ppc64le`+`s390x` :eyes:

(see also https://wiki.debian.org/LTS/)